src/containers/savedCities.container.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { citiesActions } from '../actions';
+import { CityListItemComponent } from '../components';
+
+class SavedCitiesContainer extends React.Component {
+    componentDidMount() {
+		const { getCities } = this.props;
+        getCities();
```

Comment:
I would rather call it `fetchCities`. `getCities` is a name more appropriate for the selector.

Reply:
+